ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -51,3 +52,20 @@ def prod(input, dim=None, keepdim=False, *, dtype=None, out=None):
 
 def var(input, dim, unbiased, keepdim=False, *, out=None):
     return ivy.var(input, axis=dim, correction=int(unbiased), keepdims=keepdim, out=out)
+
+
+def min(input, dim=None, keepdim=False, *, out=None):
+    if dim is None:
+        ret = ivy.min(input, axis=dim, keepdims=keepdim, out=out)
+    else:
```

Comment:
this can be changed to
```
elif out is not None:
    ...
else:
    ...
No need for the double indentation. Also some arrays such as `array(False)` might boolean evaluate to `False` so it's best to make `is not None` explicit.